source/_integrations/ecoforest.markdown

```diff
@@ -44,4 +44,11 @@ The Ecoforest integration exposes a number entity for the device power level.
 
 ## Sensor
 
-The Ecoforest integration exposes a sensor for the current ambient temperature
+The Ecoforest integration exposes multiple sensors to monitor various features:
+
+- temperature: a sensor for the current ambient temperature
+- cpu_temperature: a sensor for the current CPU temperature
+- gas_temperature: a sensor for the current gas temperature
+- ntc_temperature: a sensor for the current <abbr title="negative temperature coefficient">NTC</abbr> probe temperature
+- status: a sensor for the current status of the device
+- alarm: a sensor for the current alarm of the device
```

Comment:
```suggestion
- alarm: a sensor for the current alarm of the device. Possible values are: `air depression`, `pellets`, `cpu overheating`, `unknown`.
```

Reply:
Can we add the possible values? Or is there a particular reason you did not list them?

Reply:
we can thanks.